src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md

```diff
@@ -9,280 +9,39 @@ ingress:
-
-  If you are using Mapbox instead of Google Maps
-
-- **`REACT_APP_SHARETRIBE_MARKETPLACE_CURRENCY`**
-
-  The currency used in the Marketplace as ISO 4217 currency code. For
-  example USD, EUR, CAD, AUD, etc.
-
-- **`REACT_APP_CANONICAL_ROOT_URL`**
-
-  Canonical root URL of the marketplace. Remove trailing slash from the
-  domain.<br />E.g. _`https://your.domain.com`_
-
-- **`NODE_ENV`**
-
-  Node environment is used to build the app. Use 'development' for
-  development and 'production' for production.<br/> Use value:
-  'production'
-
-- **`REACT_APP_ENV`**
-
-  A more fine-grained env definition than `NODE_ENV`. For example, this
-  is used to send environment info to logging service: Sentry. (If you
-  have enabled it with `REACT_APP_SENTRY_DSN`).<br/> Use value:
+## What is a production environment and how does it differ from a test environment?
+
+- test environment => meant for testing and exploring, no real user data
+  or transactions
+  - in Flex: linked to the Flex **dev** environment i.e. uses the
+    application info from that environment
+  - deployment can be e.g. on a free plan with shared resources since
+    there's no requirement for consistent uptime for a substantial
+    volume of users
+- production environment => meant for the real-life usage of the
+  application, and moving code from test to production happens once the
+  code has been sufficiently tested first
+
+  - in Flex: linked to the Flex **prod** environment
+  - deployment should be on a paid plan with dedicated resources
+
+- Read more about [Flex environments](/concepts/flex-environments/)
+
+## Where to host your production client application?
+
+- Heroku, render, aws
+- What to consider when hosting
+- Why can't ftw be hosted on serverless platforms such as Vercel or
+  Netlify
+  - server side rendering
+  - possible to refactor server features to lambda functions for
+    serverless hosting, however it is likely not worth the effort vs the
+    benefit since there are also options for hosting the server
+
+## Launching to production
+
+- Some environment vars need to be changed, such as Set NODE_ENV to
   'production'
-
-- **`REACT_APP_SHARETRIBE_USING_SSL`**
-
-  Redirect HTTP to HTTPS?<br/> Use value: true
-
-- **`SERVER_SHARETRIBE_TRUST_PROXY`**
-
-  Set this when running the app behind a reverse proxy, which is how
-  Heroku works.<br/> Use value: true
-
-- **`REACT_APP_CSP`**
-
-  Content Security Policy (CSP). Read more from
-  [this article](/ftw/how-to-set-up-csp-for-ftw/).<br /> Accepts values:
-  _block_ and _report_. <br/> Use value: _block_.
-
-- **`REACT_APP_AVAILABILITY_ENABLED`**
-
-  Possible values: true/false
-
-- **`REACT_APP_DEFAULT_SEARCHES_ENABLED`**
-
-  Possible values: true/false
-
-> **NOTE:** If you change these variables, you need to deploy the app
-> again.
-
-![Heroku settings](./heroku-config-vars.png)
-
-### Set up domains and certificates
-
-Heroku manages SSL certificates automatically for new applications. You
```

Comment:
The Render tutorial doesn't contain any info on domain and certificate setup => we need to reference these somewhere (e.g. https://render.com/docs/tls and https://render.com/docs/custom-domains)